services/core-web/src/tests/components/mine/ExplosivesPermit/__snapshots__/MineExplosivesPermitTable.spec.js.snap

```diff
@@ -1,323 +1,7 @@
-            "height": 2,
-            "latitude": null,
-            "length": 1,
-            "longitude": null,
-            "quantity": 12,
-            "tag_no": "1",
-            "type": "DET",
-            "type_no": "1",
-            "width": 2,
-          },
-          Object {
-            "construction": "string",
-            "distance_dwelling": 60,
-            "distance_road": 70,
-            "height": 2,
-            "latitude": null,
-            "length": 1,
-            "longitude": null,
-            "quantity": 12,
-            "tag_no": "1",
-            "type": "DET",
-            "type_no": "1",
-            "width": 2,
-          },
-        ],
-        "documents": Array [],
-        "expiry_date": "2007-12-04",
-        "explosive_magazines": Array [
-          Object {
-            "construction": "string",
-            "distance_dwelling": 60,
-            "distance_road": 70,
-            "height": 2,
-            "latitude": null,
-            "length": 1,
-            "longitude": null,
-            "quantity": 12,
-            "tag_no": "1",
-            "type": "EXP",
-            "type_no": "1",
-            "width": 2,
-          },
-          Object {
-            "construction": "string",
-            "distance_dwelling": 60,
-            "distance_road": 70,
-            "height": 2,
-            "latitude": null,
-            "length": 1,
-            "longitude": null,
-            "quantity": 12,
-            "tag_no": "1",
-            "type": "EXP",
-            "type_no": "1",
-            "width": 2,
-          },
-        ],
-        "explosives_permit_guid": "81324623978135",
-        "isExpired": true,
-        "issue_date": "2007-12-04",
-        "issuing_inspector_name": "John",
-        "issuing_inspector_party_guid": null,
-        "key": "81324623978135",
-        "latitude": null,
-        "longitude": null,
-        "mine_operator_guid": null,
-        "mine_operator_name": "Mike",
-        "mines_permit_number": "P-39039404",
-        "now_number": "1500615-2021-12",
-        "permit_number": "BC-145411",
-        "source": "Core",
-      },
-    ]
-  }
-  expandProps={
-    Object {
-      "getDataSource": [Function],
-      "recordDescription": "document details",
-      "rowExpandable": [Function],
-      "rowKey": [Function],
-      "subTableColumns": Array [
-        Object {
-          "dataIndex": "explosives_permit_document_type_code",
-          "key": "explosives_permit_document_type_code",
-          "render": [Function],
-          "title": "Category",
-        },
-        Object {
-          "dataIndex": "document_name",
-          "key": "document_name",
-          "render": [Function],
-          "title": "Document Name",
-        },
-        Object {
-          "dataIndex": "upload_date",
-          "key": "upload_date",
-          "render": [Function],
-          "title": "Date",
-        },
-      ],
-    }
-  }
-  rowKey={[Function]}
-/>
+<ContextConsumer>
```

Comment:
I don't know if it really matters since we're moving towards more cypress testing.  But to get nested components to generate the correct snap I've had better luck with `@testing-library/react` rather than enzyme.  

There's a good example here.
`services/core-web/src/tests/components/common/FileUpload.spec.tsx`

That said, maybe it's not worth bothering.  Up to you.

Reply:
DONE.
I didn't like the especially-useless snap, but it took some fiddling because it's a withRouter component.